test/test_jit.py

```diff
@@ -59,17 +59,18 @@ def test_scriptmodule_Spectrogram(self):
 
     def test_torchscript_create_fb_matrix(self):
         @torch.jit.script
-        def jit_method(n_stft, f_min, f_max, n_mels):
-            # type: (int, float, float, int) -> Tensor
-            return F.create_fb_matrix(n_stft, f_min, f_max, n_mels)
+        def jit_method(n_stft, f_min, f_max, n_mels, sample_rate):
+            # type: (int, float, float, int, int) -> Tensor
```

Comment:
I see `sample_rate` being converted to float later. Does it need to be an integer instead of a float?

Reply:
I guess technically you could sample at a non integer rate per second but I've never seen this before - typically audio is always 8kHz, 44.1kHz, etc..

Reply:
In any case, this would be a discussion for elsewhere :) `sample_rate` is consistently an `int` elsewhere.